app/uk/gov/hmrc/gform/sharedmodel/formtemplate/destinations/Destinations.scala

```diff
@@ -47,4 +47,17 @@ object Destinations {
       safeCast[DestinationList, Destinations](destinationListFormat) orElse
         safeCast[DestinationPrint, Destinations](destinationPrintFormat)
     )
+
+  implicit val leafExprs: LeafExpr[Destinations] = new LeafExpr[Destinations] {
+    def exprs(path: TemplatePath, t: Destinations): List[ExprWithPath] = t match {
+      case DestinationList(destinations, acknowledgementSection, declarationSection) =>
```

Comment:
destinations seem to have expressions like customerId, personalisation,  includeIf, which referer to form component ids. Are we not addressing them?

Reply:
We can't help with typos in handlebar controlled snippets for example mentioned includeIf.
As for personalisation I think we are now using that destination at the moment... 
So only thing worth to support is customerId

Reply:
You meant not using personalisation?

Reply:
Not using `Email` destination